python/tk_desktop/grouping_list_view.py

```diff
@@ -114,7 +114,7 @@ class GroupingListView(ActionListView):
     expanded_changed = QtCore.Signal(str, bool)
 
     def __init__(self, parent=None):
-        QtGui.QListView.__init__(self, parent)
+        ActionListView.__init__(self, parent)
```

Comment:
I feel like we're in an habit of always calling the base class directly, especially when dealing with Qt based classes. From my understanding super is useful when dealing with multiple inheritance, which is not the case here.